static/current-version.js

```diff
@@ -1,2 +1,2 @@
-export const currentVersion = 'currentVersion';
-export const nextVersion = 'nextVersion';
+export const currentVersion = '3.3.0';
+export const nextVersion = undefined;
```

Comment:
This does not need to be adapted, the file is automatically touched during the build process.

Reply:
makes sense, I'll revert it